presto-native-execution/src/test/java/com/facebook/presto/nativeworker/HiveExternalWorkerQueryRunner.java

```diff
@@ -15,17 +15,24 @@
 
 import com.facebook.airlift.log.Logger;
 import com.facebook.airlift.log.Logging;
+import com.facebook.presto.testing.QueryRunner;
 import com.facebook.presto.tests.DistributedQueryRunner;
 
-public class HiveExternalWorkerQueryRunner
-{
-    private HiveExternalWorkerQueryRunner() {}
+public class HiveExternalWorkerQueryRunner {
+    private HiveExternalWorkerQueryRunner() {
+    }
+
     public static void main(String[] args)
-            throws Exception
-    {
-        // You need to add "--user user" to your CLI for your queries to work
+            throws Exception {
+        // You need to add "--user user" to your CLI for your queries to work.
```

Comment:
Formatting changed - is it right formatting for java?

Reply:
It's not it will fail format check

Reply:
Ah... I regret doing global formatting..

Reply:
> Ah... I regret doing global formatting..

Where did you perform the global formatting? In CLion or IJ?
IJ does not format java files like c++.